docs/updating-the-docs.md

```diff
@@ -48,3 +46,26 @@ Examples are included in the markdown files using the [pymdown snippet notation]
 Make sure you commit both the changes to the `examples` and `databuilder/snippets` directories.
 
 If you are updating multiple examples it might be easier to the extraction step happen each time you change the python modules, you can do this with `just databuilder/watch-examples`.
+
+
+## Updating Data Builder backend and contract documentation
+
+!!! note "You will need both Python 3.7 and 3.10 on your PATH to make these updates"
```

Comment:
suggestion:

* Maybe clarify briefly why you need both (linking to #493 might suffice)
* Maybe worth warning that this isn't supported for Windows as yet, if that's the case still. [^1]

[^1]: We could try Gitpod, but not sure how this would work out.

Reply:
The Justfile seems bash-heavy. My assumption is that trying to run the same commands in Windows will fail.